ruma-federation-api/src/user.rs

```diff
@@ -0,0 +1 @@
+pub mod get_devices;
```

Comment:
This new module needs to added to ruma-federation-api via a `pub mod user;` statement in `lib.rs`.